index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <p id="logo"> My Logo </p>
+        <img id="humburger-btn" src="images/icons8-hamburger-menu-64.png" alt="hamburger btn"/>
+    </header>
+    <main class="container">
+        <section class="first-section">
+            <h2>I’m Crispin<br/> Glad to see<br/> you!</h2>
```

Comment:
- [ ] This should be an `h1` element because it is semantically correct and for SEO reasons. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.